src/overlays/actors/ovl_Bg_Tobira01/z_bg_tobira01.h

```diff
@@ -5,9 +5,17 @@
 
 struct BgTobira01;
 
+typedef void (*TobiraActionFunction)(struct BgTobira01*, PlayState*);
+
 typedef struct BgTobira01 {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x2C];
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ TobiraActionFunction actionFunction;
+    /* 0x160 */ s16 doorYPositionTick;
+    /* 0x162 */ s16 isNight;
+    /* 0x164 */ f32 yPosition;
```

Comment:
```suggestion
    /* 0x160 */ s16 posYTick;
    /* 0x162 */ s16 isNight;
    /* 0x164 */ f32 posY;
```
we tend to abbreviate position as `pos` and the direction comes afterwards. Also, the `door` part comes as part of the struct, so I don't think it's needed (or otherwise put door in front of both `posY` and `posYTick`. Same would apply to `tempDoorYPosition`